.storybook/manager.js

```diff
@@ -1,5 +1,6 @@
 import { addons } from '@storybook/manager-api';
 import { create } from '@storybook/theming';
+import { enhanceStoryName } from '../src/stories/lib/story-names';
```

Comment:
You may skip all component files (batch modifications) but please do review the `story-names` file :wink: